src/Cart.js

```diff
@@ -2,16 +2,22 @@ class Cart {
 
   constructor() {
     this.totalSum = 0;
+    this.items = [];
   }
 
   add(product) {
+    this.items.push(product);
     this.totalSum += product.price;
```

Comment:
We don't actually need to store the running totalSum each time. In fact, doing so is a risk because someone might implement remove() and forget to update it. :thinking:

Reply:
Fixed in ad991db3d229899fa46305619c69c0a0d8318b36.